tests/test_agent_jules_on_issues.py

```
@@ -0,0 +1,91 @@
+                block,
+            )
+
+    def test_secret_availability_and_failed_invocation_take_bounded_paths(self) -> None:
+        self.assertNotIn("if: ${{ secrets.JULES_API_KEY", self.workflow)
+        for name in ("jules-on-label", "jules-on-mention"):
+            block = self.job_block(name)
+            self.assertIn("name: Detect Jules API key availability", block)
+            self.assertIn("id: jules-api", block)
+            self.assertIn("continue-on-error: true", block)
+            self.assertIn("steps.jules-api.outcome == 'failure'", block)
+            self.assertIn("Fallback Jules App request when API key is absent or invocation fails", block)
```

Pylint (4.0.6)

[convention] Line too long (105/100)

(C0301)